android/src/main/java/com/inbrain/InBrainSurveysModule.java

```diff
@@ -45,15 +45,10 @@
 public class InBrainSurveysModule extends ReactContextBaseJavaModule implements InBrainCallback {
 
     private final ReactApplicationContext reactContext;
-    private HashMap sessionData;
```

Comment:
Looks like you forgot about `public void setSessionID(final String sessionId)` and `public void setDataOptions(final ReadableMap data)` methods - they still uses those variables. Please, update the methods.
And do not forget to run the app at both platforms (iOS and Android) to ensure all is fine...